make/tests

```diff
@@ -208,3 +208,7 @@ endif
 # Include the test dependencies
 ##
 include make/test-math-dependencies
+
+.PHONY: test 
```

Comment:
can you take this target out of the makefile? I don't think there should be a circular dependency here.